x-pack/legacy/plugins/siem/public/containers/timeline/index.tsx

```diff
@@ -79,15 +79,17 @@ class TimelineQueryComponent extends QueryTemplate<
       sourceId,
       sortField,
     } = this.props;
+    const defaultIndex =
+      indexPattern == null || isEmpty(indexPattern)
```

Comment:
`isEmpty` should also get the null and undefined case, the `indexPattern == null ||` is redundant.